meson.build

```diff
@@ -1717,6 +1717,15 @@ if conf.get('BPF_FRAMEWORK') == 1
                 '-c',
         ]
 
+        # If c_args contains prefix maps copy them along with the values, in order to avoid breaking
+        # reproducible builds
+        foreach opt : c_args
+                if opt.contains('-ffile-prefix-map=') or opt.contains('-fdebug-prefix-map=') or opt.contains('-fmacro-prefix-map=')
```

Comment:
i'd really prefer startswith.

and can we put that in a separate list? i.e. 

```
propagate_cflags = [
        "-ffile-prefix-map=",
        "-fdebug-prefix-map=",
        "-fmacro-prefix-map=",
]
```

i have the suspicion that the list of options to propagate from the regular clags to the bpf cflags is only going to grow...